css-canvas-medical/styles.css

```diff
@@ -0,0 +1,216 @@
+* {
+  font-family: Avenir, sans-serif;
+}
+
+h1,
+h2,
+h3 {
+  font-family: "Tiempos Headline", sans-serif;
+}
+
+.container {
+  background-color: white;
+  max-width: 1000px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.col-1 {
+  flex-basis: 100%;
+}
+
+.col-2 {
+  flex-basis: calc(100% / 2);
+}
+
+.col-3 {
+  flex-basis: calc(100% / 3);
+}
+
+.col-4 {
+  flex-basis: calc(100% / 4);
+}
+
+.col-8 {
+  flex-basis: calc(100% / 8);
+}
+
+.padding10 {
+  padding: 10px;
+}
+
+.get-started-btn {
+  padding: 15px 40px;
+  margin-top: 20px;
+}
+
+.padding-hero {
+  padding: 5% 8%;
+}
+
+.padding-header {
+  padding-top: 10px;
+}
+
+.blue-logo {
+  margin-top: 1rem;
+  margin-left: 5rem;
+  margin-bottom: 15px;
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.white-logo {
+  margin-top: 5rem;
+  margin-left: 5rem;
+  margin-bottom: 1rem;
+  max-width: 50%;
+  max-height: 50%;
+}
+
+.navbar {
+  background-color: white;
+  color: lightgray;
+  margin-top: 1rem;
+  margin-right: 5rem;
+  float: right;
+}
+
+.navbar a {
+  text-align: center;
+  padding-left: 10px;
+  padding-right: 10px;
+  color: gray;
+  text-decoration: none;
+}
+
+.navbar:active {
+  color: blue;
+}
+
+.vertical-nav a {
+  color: white;
+  display: block;
+  text-decoration: none;
+  padding-bottom: 15px;
+}
+
+.bottom-nav a {
+  text-decoration: none;
+  color: gray;
+  padding-right: 40px;
+}
+
+a.button {
+  color: white;
+  background-color: #2a4463;
+  border-radius: 5px;
+  border: none;
+}
+
+.button {
+  color: white;
+  background-color: #2a4463;
+  border-radius: 5px;
+  border: none;
+}
+
+hr {
+  background-color: lightgray;
+  color: lightgray;
+  border: 0 solid lightgray;
+  height: 1px;
+}
+
+.bottom-div {
+  background-color: lightgray;
+  color: lightgray;
+  border: 0 solid lightgray;
+  height: 1px;
+  margin-top: 0;
+}
+
+.align-right {
+  float: right;
+}
```

Comment:
You can avoid using `float`, I think. I didn't teach you guys about it for a reason. In fact, I remember mentioning multiple times that it is an outdated tool to avoid 😉 . You can make a `row` inside of that `col-1` element you are applying this too and use `justify-content: flex-end` to approximate the same behavior 🤓